test/utils/fixture/fixture.go

```diff
@@ -136,8 +137,12 @@ func Cooldown(ft *Fixture) {
 	if len(ft.InitialNRTList.Items) > 0 {
 		interval := 5 * time.Second
 		ginkgo.By(fmt.Sprintf("cooldown by verifying NRTs data is settled to the initial state (interval=%v timeout=%v)", interval, settleTimeout))
-		intwait.With(ft.Client).Interval(interval).Timeout(settleTimeout).ForNodeResourceTopologiesEqualTo(context.TODO(), &ft.InitialNRTList, intwait.NRTIgnoreNothing)
 		// TODO: shall we wait for the data to settle?
+		currentNrtList, err := intwait.With(ft.Client).Interval(interval).Timeout(settleTimeout).ForNodeResourceTopologiesEqualTo(context.TODO(), &ft.InitialNRTList, intwait.NRTIgnoreNothing)
+		if err != nil {
+			diff := cmp.Diff(ft.InitialNRTList.Items, currentNrtList.Items)
```

Comment:
The variable is initialized outside the `k8swait.PollImmediate` and captured by its closure, so it should always contain the most recent Get value, isn't it?

Reply:
I see the variable is declared but not initialized so it's zero-value will be nil. you are right, it will always contain the most recent value of Get/List but in case List kept failing the value will stay nil. Unless List do initialize an empty nrt-list in case of failure. I'm not sure about that, I'm interested to verify this.
Anyhow,its possibility to fail is close to zero, so we can keep it as is.

Reply:
true, but if `currentNrtList` is zero-value, then `err` will be != nil, so we clearly catch this case